CorpusSearch/ClientApp/src/routes/Newbies.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react"
+function CorpusLink({ident="",name=""}): JSX.Element {
+    if ( name.slice(0,2) != "🎥"){
+        name = "📖" + name
+    }
+    return <div>
+        <a href={"docs/" + ident}>{name}</a>
+    </div>
+}
+export const Newbies = () => {
+type DocType =  {name:string, ident:string, uploaded:string}
+const [newDocs, setNewDocs  ] 
+    = useState<DocType[]>([])
+useEffect(()=>  {
+    const fetchNewDocs = async () => {
+        await fetch("api/metadata/latest")
+            .then((res)=> {
+                if (!res.ok) {
+                    console.warn(`metadata/latest returned status: ${res.status}`)
+                }
+                return res.json()
+            }).then((json: DocType[])=> {
+                setNewDocs(json)
+        })
+    }
+    fetchNewDocs().catch((e:Error)=> console.warn(`fetchNewDocs returned error:" ${e.message}`))
+},[]
```

Comment:
What autoformatter should I be using?

Reply:
I don't know the "best" way to format this (yet), but I just manually formatted it in a way I could live with.

Reply:
Also was able to test manually by doing these separately and looking at the warnings on the console:
* changing the url to break so that res.ok was false
* throwing an error and catching it

Reply:
As noted in another comment, Rider's is /mostly/ decent, there was a couple of manual fixups required

The main issue here is indentation, which any auto-formatter should get close to perfect